docs/sdk/wallets/auth-methods/custom-auth.md

```diff
@@ -14,9 +14,9 @@ Authentication refers to confirming a users identity.  This generally involves r
 
 Authorization refers to confirming that a user is allowed to use a PKP.  Specifically, it's checking the permissions of a PKP and making sure that the user that was Authenticated is also authorized to use a PKP.  
 
-Note: Currently, an [auth signature](../../authentication/auth-sig), is **always required** to communicate with the Lit nodes and make a request to the network. It doesn't matter if you are decrypting a piece of data or calling a Lit Action, an auth sig will always be required.
+Note: Currently, a [Session Signature](../../authentication/session-sigs/intro.md), is **always required** to communicate with the Lit nodes and make a request to the network. It doesn't matter if you are decrypting a piece of data or calling a Lit Action, a Session Sig will always be required.
 
-In the case that a user doesn’t own a wallet (and therefore cannot produce a valid AuthSig), and you're using an Auth Method supported by Lit, they can present their alternative auth method to the Lit SDK which will convert it into a “compliant” AuthSig. This is documented in our [docs](overview).  If you're not using an auth method supported by Lit, then your AuthSig is not for authorization, but you must still present one for Rate Limit authentication.  In this case, the rate limit is tracked against the AuthSig, and you may wish to pay for your users requests by giving this wallet a Capacity Credit NFT.  
+In the case that a user doesn’t own a wallet (and therefore cannot produce a valid SessionSig), and you're using an Auth Method supported by Lit, they can present their alternative auth method to the Lit SDK which will convert it into a “compliant” SessionSig. This is documented in our [docs](overview).  If you're not using an auth method supported by Lit, then your SessionSig is not for authorization, but you must still present one for Rate Limit authentication.  In this case, the rate limit is tracked against the SessionSig, and you may wish to pay for your users requests by giving this wallet a Capacity Credit NFT.  
```

Comment:
I think this: `[docs](overview)` is incorrect (doesn't link to anything), but I'm not sure what it's supposed to link to

Reply:
Good catch, let's just remove the entire para. I can write something later and add it here as this effects a very small subset of the use cases